src/lib/user/user-actor-factory.ts

```diff
@@ -1,51 +1,33 @@
 import { UserActor, UserActorConfig } from "./user-actor";
-import { HashService } from "../services/hash";
-import { Pool } from "pg";
-import { dbSelectUserIdByHashedEmail } from "../data/db-users";
-import { LRUCache } from "lru-cache";
-
-export type UserActorFactoryConfig = {
-  dbPool: Pool;
-  emailHashService: HashService;
-};
+import { BarkContext } from "../bark/bark-context";
+import { UserAccountService } from "../bark/services/user-account-service";
 
 /**
  * Responsible for creating user actors
  */
 export class UserActorFactory {
-  private config: UserActorFactoryConfig;
-  private actorConfig: UserActorConfig;
-  private idCache: LRUCache<string, string>;
-
-  constructor(config: UserActorFactoryConfig, actorConfig: UserActorConfig) {
-    this.config = config;
-    this.actorConfig = actorConfig;
-    this.idCache = new LRUCache({ max: 10 });
-  }
+  constructor(
+    private args: {
+      actorConfig: UserActorConfig;
```

Comment:
Just a reminder, that we want to get rid of UserActorConfig

Reply:
We cannot remove this yet because it is part of the getParams return value, which is used by various user-actor actions. For instance, getDogStatuses, getDogProfile, getDogPreferredVets, etc. Will add a TODO to have it remove once getParams is no longer used.

Reply:
Added the comment in 412aca04